web-library-build/src/index.ts

```diff
@@ -34,30 +34,33 @@ const sourceMatch: string[] = [
 ];
 
 // Define default task groups.
-let buildTasks: IExecutable = task('build', serial(preCopy, sass, parallel(tslint, typescript, text), postCopy));
-let bundleTasks: IExecutable = task('bundle', serial(buildTasks, webpack));
-const postProcessSourceMaps: PostProcessSourceMaps = new PostProcessSourceMaps();
-const validateShrinkwrapTask: ValidateShrinkwrapTask = new ValidateShrinkwrapTask();
-const generateShrinkwrapTask: GenerateShrinkwrapTask = new GenerateShrinkwrapTask();
+export const buildTasks: IExecutable = task('build', serial(preCopy, sass, parallel(tslint, typescript, text), postCopy));
```

Comment:
i'm saying, define it above this and actually use it so you don't duplicate the definition.